src/loaders/lottie/tvgLottieModel.h

```diff
@@ -613,6 +613,28 @@ struct LottieComposition
 {
     ~LottieComposition();
 
+    LottieObject* findObject(const char* sid) {
+        auto targets = this->root->children;
+        return getObject(sid, root->children);
+    }
+
+    LottieObject* getObject(const char* sid, const Array<LottieObject*> children) {
```

Comment:
Should this method be introduced as a member of LottieComposition()? I think this is not relevant to it. Probably you can add this as a just static helper function (see 29 line in tvgLottieModel.cpp )

Also, Just for your information, Please double-check the difference in binary size between inline (.h) and (.cpp) implementations when you adding a inline function. We will use the one with the smaller size.